packages/sql/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@carto/toolkit-sql';
```

Comment:
These declar should not be necessary, could you try removing them? And the references to index.d.ts

Reply:
I have removed them, and it compiles. But it complains on npm run test